tests/tests_fabric/conftest.py

```diff
@@ -60,6 +60,8 @@ def restore_env_variables():
         # set by XLA FSDP on XRT
         "XRT_TORCH_DIST_ROOT",
         "XRT_MESH_SERVICE_ADDRESS",
+        # set by torchdynamo
+        "TRITON_CACHE_DIR",
```

Comment:
Should we delete this if it exists on teardown of the test?

Reply:
In my head, the intention of this conftest was that we make sure that any environment flags set by Lightning are cleaned up. If you follow this definition, you could ignore this one since it's set by Inductor.

I could be persuaded though.